test/val/val_decoration_test.cpp

```diff
@@ -548,4 +548,159 @@ OpFunctionEnd
                         "Component decorations"));
 }
 
+TEST_F(ValidateDecorations, RuntimeArrayOfDescriptorSetsIsAllowed) {
```

Comment:
These tests are long enough that it would be useful to add the GLSL code in comments for them.

Reply:
done